tests/Transaction/Factory/InputSignerTest.php

```diff
@@ -85,7 +85,8 @@ public function getVectors()
     public function testInvalidSolveSignData($description, EcAdapterInterface $ecAdapter, TransactionInterface $tx, TransactionOutput $txOut, SignData $signData, $exception, $exceptionMsg)
     {
         try {
-            new InputSigner($ecAdapter, $tx, 0, $txOut, $signData);
+            (new InputSigner($ecAdapter, $tx, 0, $txOut, $signData))
+                ->extract();
```

Comment:
This is the one of the bigger side effects here - extract needs to be run before the instance is safe to be used.